docs/core/integration-quality-scale/rules/unique-config-entry.md

```diff
@@ -17,6 +17,8 @@ This can lead to duplicated devices and entities with unique identifiers collidi
 Any discovery flow must also ensure that a config entry is uniquely identifiable, as otherwise, it would discover devices already set up.
 
 To prevent this, we need to ensure that the user can only set up a device or service once.
+Trying to set up a device or service that is already set up should be prevented by the integration.
+If an update to the configuration is required, it should be handled by the reconfiguration flow instead, as it allows us to be more consistent with what gets updated.
```

Comment:
Yes, though I also learned that means my english is sloppy and less precise :)

Given that caveat, this is how I understand it:
```suggestion
Updates to an existing device or service configuration are only allowed through the reconfiguration flow. Updates to authentication credentials are allowed in a reauthentication flow.  Otherwise, trying to set up a device or service that is already set up should be prevented by the integration.
```